src/components/HighlightedLink.js

```diff
@@ -1,6 +1,43 @@
+import PropTypes from "prop-types";
 import { Link } from "gatsby";
+import { useLocation } from "react-use";
 import styled from "styled-components";
 
-export default styled(Link).attrs({
+const StyledLink = styled(Link).attrs({
   className: "text-primary underline-offset-2 decoration-1 decoration-dotted hover:text-primary-light hover:underline",
 })``;
+
+/**
+ * This an extended version of Gatsby's <Link /> component, which
+ * is also styled to match Skynet Labs' dashboard's theme.
+ */
+export default function HighlightedLink({ persistQueryString, to, ...props }) {
+  const { search, origin } = useLocation();
+  let url = to;
```

Comment:
nit: why not just name the input `url`?

Reply:
oh is it because the `to` param is a Gatsby link param?

Reply:
Yeah, exactly :)